models/preprocessing.py

```diff
@@ -87,7 +124,10 @@ def glob_to_snapshot(top_level_path):
 
     # Labels
     prefix = top_level_path + "/Labels/"
-    files = glob.glob(prefix + "*/*")
+    if extra_level:
+        files = glob.glob(prefix + "Mimic3/*/*")
```

Comment:
Ah if you just put in Mimic3 in the top level path it'll work. oh well not a huge deal.